### ISSUE TITLE
Transfer ownership of Discord.js Documentation

### ISSUE THREAD

**Issue by amishshah** [rtfd/readthedocs.org#3208]
## Details

* Project URL: http://discordjs.readthedocs.io/en/latest/index.html
* Read the Docs username (if applicable): hydrabolt

Hello, I'm the maintainer of the [discord.js](https://discord.js.org/) project and a while back, we used to use readthedocs to write our documentation. We then switched to using JSDoc for our code, but we're again thinking of switching back to readthedocs to maybe write some guides and tips for users.

In this time period, user @Macdja38 used the documentation site to host documentation for his fork of our project, which seems to only be in use by himself (at this moment in time). I've asked him multiple times to rename the site or move the documentation elsewhere because it is confusing our users who stumble onto the old documentation and then ask for support using the old documentation in our support server; looking at my chat history with him, it seems the first time I've asked him to make a change was the 21st April.

While he has made some effort to point users to the new documentation, his redirect links only work on the homepage, so it's not much use.

I believe I have tried to be cooperative with macdja and I've been asking him for several months to consider renaming the project so we can reclaim the project URL for the official development of discord.js, but it seems like this is not going to happen. I'd like to point out that my copyright is also on the bottom of the current readthedocs documentation.

If you could help, I'd like a transfer of the http://discordjs.readthedocs.io/ project's ownership to myself so we can put it to better use and also back to the library's official developers' hands.

Thanks

**Comment by hydrabolt** [comment]
Any update on this?

**Comment by safwanrahman** [comment]
@hydrabolt Can you go through the DCMA takedown policy?
https://docs.readthedocs.io/en/latest/dmca/index.html

**Comment by iCrawl** [comment]
Hello readthedocs team,

I certainly do not want to be the booman and call out a sloppy way of handling this issue, but looking at the date of this issue and follow-up email conversations, which are all available if needed, I have to ask myself: does it even matter?
We followed the procedure inquired and filled out a DMCA takedown to the old doc site (http://discordjs.readthedocs.io/en/latest/) which **WE** (the maintainers of the project) do not have access to in any kind of way. You state as follows:

>The author of the content in question will be asked to make changes to the content specified in the takedown request. The author will have 24 hours to make these changes. The copyright holder will be notified if and when this process begins

Additionally:
>The author must notify Read the Docs that changes have been made within 24 hours of receiving a takedown request. If the author does not respond to this request, the default action will be to disable the Read the Docs project and remove any hosted versions

We waited a solid **2** months before sending another email asking what the status to the takedown is. The only response we got was: "We are still waiting for contact and resolution here [...]"
Now I have to ask, **what** are we waiting on? The DMCA takedown policy clearly states **24** hours. This is an issue that has been dragging on for over 2 **months**.

Adding to that there has even been another issue created #3133 that is referring to our doc site, hosted by your service, which we have no control over. In this issue you linked (http://docs.readthedocs.io/en/latest/abandoned-projects.html) the article to abandoned projects which states as follows:

>- Owner is unreachable (see Reachability)
>- The project has no proper documentation being served (no successful builds) or does not have any releases within the past twelve months
>- No activity from the owner on the project’s home page (or no home page found).

Now I do agree that the second point mentioned might not be entirely true, since there are docs hosted on it, but they are majorly outdated for a version **we** the maintainers do not support anymore. We have no access to the docs site, let me remind you of that fact. We cannot change it, we cannot take it down, we simply cannot do anything but watch how our users get access to documentation for our library, that is outdated by several months, via a simple Google search.

After 7 months of trying to reach the *new* author of **our** docs page and trying to resolve this issue, we are no further, so we decided to reach out to readthedocs to maybe get our issue resolved by a higher authority. But after issuing a DMCA takedown and two more months we are no further than we were before.

I would gladly engage into a conversation regarding this issue or even better just showing compliance to the DMCA takedown policy.

Thank you.

**Comment by stsewd** [comment]
Does https://github.com/macdja38/discord.js/pull/8 solves the issue? 

All the pages https://discordjs.readthedocs.io/en/latest/index.html now have this warning

![screenshot-2018-4-16 welcome to discord js s documentation discord js 8 2 0 documentation](https://user-images.githubusercontent.com/4975310/38799233-83ca74ac-4129-11e8-993d-2336f290124f.png)

**Comment by iCrawl** [comment]
No it doesn't.

The code in the branch he forked is not maintained anymore and will in most cases probably not even work anymore. Everything about this "branch" (which still resides in our repository btw, but then the docsite got snatched away from his fork) is not maintained anymore and potentially risky to use.

So no, we still would like to have full control over the docs site which is ours in the first place any way, and not some person who barely maintains a fork that is effectively harming is because he refuses to cooperate with us, but also with you. Again I would like you to look at the date of this issue, not to mention the emails that got sent forth and back between you, us the maintainers and the aggressor.
There's clear evidence that this is harmful behaviour and I for god knows do not understand how it is "impossible" to restore a docsite to the rightful owner of a project, compared to a fork.

**Comment by stsewd** [comment]
ok, sorry for the late response. I think @humitos may help to clarify some things here.

**Comment by ericholscher** [comment]
Sorry for the slow response here. We currently don't have a policy in place that lets us take a project name away from an active user of that project on Read the Docs.

It sounds like we should have a policy in place, and from the research I've done, generally people will do this when the name in question is an owned trademark. That is the only way to really prove ownership, especially since we support multiple ecosystems with often conflicting namespaces. Do y'all have the name `discord.js` trademarked?

**Comment by devsnek** [comment]
as neither of the two parties have filed for a trademark, they are under first-to-use trademark law, which means @hydrabolt holds the trademark as @macdja38's project and docs are a fork of @hydrabolt's work under the same name.

**Comment by hydrabolt** [comment]
I think we're regressing here. We've already established that the user is not active and his project is not active.

I first emailed you guys in October, and I have been told 2 weeks ago that you guys are in the process of renaming the docs as suggested, an update on this would be nice.

**Comment by agjohnson** [comment]
I've already been in touch with everyone and we have a solution, I need to finalize the cutover this week.

the discordjs project is going to become discordjs-v8, as the project is named in the docs. discord the namespace will free up afterwards.

**Comment by stsewd** [comment]
The https://discordjs.readthedocs.io project is deleted, can we close this issue?

**Comment by amishshah** [comment]
Yep, sorry this has been resolved for a while now, will close
